tests/roots/test-ext-autodoc/target/typehints.py

```diff
@@ -1,7 +1,13 @@
-from typing import Any, Tuple, Union
+import pathlib
+from typing import Any, Tuple, TypeVar, Union
 
 CONST1: int
+#: docstring
```

Comment:
I added these docstrings because the items are not picked up by autodoc otheriwse.

Reply:
Ah, that's the long-standing #1063.